custom-recipes/deeplearning-image-extract-v2/recipe.py

```diff
@@ -1,9 +1,11 @@
 import pandas as pd
-import dku_deeplearning_image.utils as utils
 from dataiku.customrecipe import get_recipe_config
 
+import dku_deeplearning_image.utils as utils
+import dku_deeplearning_image.constants as constants
```

Comment:
I think we should change `constants` to `dku_constants` to avoid some annoying pb later on with py37

Reply:
Will open a new PR to fix the naming of different files. I'll keep that in mind